applications/app/controllers/CampaignsController.scala

```diff
@@ -0,0 +1,47 @@
+package controllers
+
+import conf.Configuration
+import common._
+import model._
+import play.api.libs.ws.{WSClient, WSResponse}
+import play.api.mvc._
+import play.api.libs.json._
+
+import scala.concurrent._
+
+class CampaignsController(
+  val controllerComponents: ControllerComponents,
+  val wsClient: WSClient,
+)(implicit context: ApplicationContext)
+  extends BaseController with ImplicitControllerExecutionContext with Logging {
+
+  val root = Configuration.formstack.url + "/form/"
+  val endpoint = "/submission.json"
+  val token = Configuration.formstack.editorial.oAuthToken
```

Comment:
I was thinking about that. It's probably a good idea to have both the old and the new config format in the same file when this PR goes lives, and then remove the old format. That's because the config must be updated before this is merged.

Reply:
Do you mean we should leave the old config format in S3?